extensions/notebook/src/book/bookVersionHandler.ts

```diff
@@ -38,7 +38,7 @@ export function convertFrom(version: string, section: JupyterBookSection): Jupyt
 			divider: (section as IJupyterBookSectionV1).divider,
 			header: (section as IJupyterBookSectionV1).header,
 			external: (section as IJupyterBookSectionV1).external,
-			numbered: !(section as IJupyterBookSectionV1).not_numbered,
+			numbered: (section as IJupyterBookSectionV1).not_numbered ? !(section as IJupyterBookSectionV1).not_numbered : undefined,
```

Comment:
You're only going to be setting numbered if not_numbered is true - is that intentional? The other conversions you do take into account both true and false.

Reply:
I don't believe this is intentional; if `not_numbered` was explicitly set to `false` previously, we should respect that. What do you think @barbaravaldez

Reply:
yup, is not intentional